lib/cisco_node_utils/vxlan_global.rb

```diff
@@ -29,17 +30,19 @@ def initialize(instantiate=true)
     end
 
     def enable
-      config_set('vxlan_global', 'feature', state: '')
+      Feature.fabric_enable unless Feature.fabric_enabled?
```

Comment:
I made some changes to feature.rb as part of my PR. One thing I'm doing is removing the `unless Feature._enabled?` from the callers and moving them into the feature setters; ie. above would be:

```
    Feature.fabric_enable
```

and feature.rb would have:

```
    def self.fabric_enable
      return if Feature.fabric_enabled?
      config_set('feature', 'fabric')
    end
```